src/ui/modal.svelte

```diff
@@ -1,47 +1,40 @@
 <script lang="ts">
-  export let dark = false;
```

Comment:
There's no more a dark and light modal, each one of them can be customised with custom colors.